committee-code-of-conduct/charter.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes Code of Conduct Committee Charter
+
+## Mission/Purpose
+Our primary mission is creating and maintaining a safe and respectful community.
+Our role is to provide and enforce a well-considered viewpoint on what
+constitutes acceptable behavior within our community. The [Code of
+Conduct](https://git.k8s.io/community/code-of-conduct.md) serves as the primary
+policy document and is supported with additional references and tools as needed.
+Since maintaining a safe environment is a very large part of what the committee
+does, we must carefully balance transparency of process with preserving the
+privacy of those who come to the committee or other community leaders with an
```

Comment:
I think the code of conduct committee should also consider the privacy of those accused of causing the incident.

Reply:
We've discussed and agree this language should be more broad. We do say, further down, that we intended the privacy to be for everyone, including the accused.

We will send a followup PR (after this merges) to discuss and address this specific area.